src/components/Pressable.re

```diff
@@ -7,14 +7,9 @@ external rippleConfig:
   (~color: Color.t=?, ~borderless: bool=?, ~radius: float=?, unit) =>
   rippleConfig;
 
-type interactionState = {
-  pressed: bool,
-  // React Native Web
-  hovered: option(bool),
-  focused: option(bool),
-};
-
-[@react.component] [@bs.module "react-native"]
+[@deprecated "Please use Pressable_ instead"]
```

Comment:
```suggestion
[@deprecated "Please use `Pressable_` instead of `Pressable` to benefit of the full benefit of Pressable component (eg: pressed state in style props and children function). In next major release (0.64), `Pressable_` will become `Pressable`."]
```